TidepoolOnboardingKitUI/Support/PreferredGlucoseUnit.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PreferredGlucoseUnit.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 1/29/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKitUI
+
+public class PreferredGlucoseUnit: ObservableObject {
```

Comment:
[nit] this is just a suggestion, but I'm wondering if we don't want to start calling things like this "Observables" (i.e. consider changing the name to `PreferredGlucoseUnitObservable`)

Reply:
Before I make this change, I'd like to get 100% agreement from the rest of the team that this will be our "standard" going forward.

Reply:
Added to dev meeting page for next week: https://tidepool.atlassian.net/wiki/spaces/TE/pages/1150845361/Dev+Sync+Meeting+2021-03-11

I will be back in this code for a while so I can address it after a decision is made by the team. Deferring for now.